models/cobalt/model.sdf

```diff
@@ -168,6 +173,75 @@
       </velocity_decay>
     </link>
 
+    <link name='hydrophones'>
+      <pose> 0 0 0 0 0 0 </pose>
+      <inertial>
+        <mass> 0.001 </mass>
+      </inertial>
+      <visual name='ref_hydrophone_visual'>
```

Comment:
If we decide to load positions from a settings file, it would be good to have the visual. But we could make them smaller, not need for them to be huge.